src/nph.nim

```diff
@@ -10,7 +10,18 @@ import
 
 import "$nim"/compiler/idents
 
-import std/[parseopt, strutils, os, sequtils]
+import std/[parseopt, strutils, os, sequtils, terminal]
+import pkg/hldiffpkg/edits
+import pkg/adix/lptabz
+
+# Silence LPTabz warnings by redirecting to /dev/null
```

Comment:
why is this needed? lptabz warnings are not enabled by default afaict: https://github.com/c-blake/adix/blob/f5c4e63dae283b3dd5ea358c8fe920876a0cff14/adix/lptabz.nim#L85